src/components/InRequest/InRequest.tsx

```diff
@@ -57,6 +75,11 @@ export const InRequest: FunctionComponent<IInRequestComp> = (props) => {
   /* Hook to update this request */
   const updateRequest = useUpdateRequest(props.request.Id);
 
+  /** State keepeing track of which type of update is currently processing */
+  const [updateType, setUpdateType] = useState<"cancel" | "complete" | "none">(
+    "none"
+  );
+
```

Comment:
Nice.  Removed the uneeeded state.  Did add comment to new const that we have to pair it with data from updateRequest since it will now default to "complete" if the cancel dialog is closed.